### ISSUE TITLE
View timeline animation is broken when I refresh the page.

### ISSUE THREAD

**Issue by finkusuma-dev** [flackr/scroll-timeline#135]
I have this simple setup to test view timeline animation. If I scroll all the way to the bottom and then refresh the page, the animation is broken.

![animation.gif](https://media3.giphy.com/media/v1.Y2lkPTc5MGI3NjExc2V6ZThuYWQyaDBicjJyMTZ5ZGt5Z2Rkc29keHEwanRvazBjN254aCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/mKHxCjhRwT4f6CcSER/giphy.gif)

[Gist](https://gist.github.com/finkusuma-dev/dfb3edaa306facd7c612be5347c6e3c4)

animate.html
```html
<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Animate Using ViewTimeline</title>

  <link rel="stylesheet" href="style.css"/>  
  <script src="./animate.js" type="module" defer></script>
</head>

<body>
  <!-- Using ViewTimeline -->
  <div class="fixed">
    <div class="">Using ViewTimeline</div>
  </div>
  <div class="box-1"></div>

  <div id="box-animated" class="box-animated"></div>
  <div class="box-2"></div>
  <div class="box-2"></div>

</body>

</html>
```

animate.js
```javascript
import 'https://flackr.github.io/scroll-timeline/dist/scroll-timeline.js';
/// animated one box
const animatedBox = document.querySelector('#box-animated');

animatedBox.animate(
  {
    backgroundColor: ['red', 'darkred'],
    // width: ['0', '100%'],
    transform: ['scaleX(0)', 'scaleX(1)'],
  },
  {
    fill: 'both',
    timeline: new ViewTimeline({
      subject: animatedBox,
      source: document.documentElement,
    }),
    rangeStart: { rangeName: 'contain', offset: CSS.percent(0) },
    rangeEnd: { rangeName: 'contain', offset: CSS.percent(100) },
  }
);
```

style.css
```css
* {
      margin: 0;
      padding: 0;
      box-sizing: border-box;
    }

    .fixed {
      position: fixed;
      top: 0;
      left: 0;
      right: 0;
      padding: 0 0 5px;
      background-color: #ffff0077;
    }

    .box-1 {
      width: 100%;
      height: 800px;
      background-color: rgb(255, 214, 160);
    }

    .box-2 {
      width: 100%;
      height: 700px;
      background-color: rgb(255, 214, 160);
    }

    .box-animated {
      width: 100%;
      height: 100px;
      background-color: hsl(0, 100%, 90%);
      transform-origin: left;
    }
```

**Comment by finkusuma-dev** [comment]
It looks okay now 👍

![view timeline okay](https://i.giphy.com/media/IWew9pcS51OB9jaaAo/giphy.webp)

**Comment by bramus** [comment]
Nice, looks like the fix in #145 also tackled this one.
